core/infra/virtual_machine/jvm/java_virtual_machine.cpp

```diff
@@ -40,7 +39,7 @@ namespace jvm {
     std::unique_ptr<JavaContext> initializeVM(const std::string& packageNameUnderInstances, const std::string& contractName) {
 
         const auto IrohaHome = []() {
-            const auto p = getenv("IROHA_HOME");
+            const auto p = config::get_iroha_home();
             if (p == nullptr) {
                 throw "[FATAL] Set environment $IROHA_HOME";
```

Comment:
1. This shows:
```
terminate called after throwing an instance of 'char const*'
Aborted
```
`throw` needs an exception class in order to show message correctly. (actually, this is my mistake. sorry ;(

2. I think it is better to include this nullptr check into `config::get_iroha_home()`. If this change is applied, all variables such as `IrohaHome` can be removed and replaced with `config::get_iroha_home()`, and in that case, exception message `"[FATAL] Set environment $IROHA_HOME"` is better.